Makefile

```diff
@@ -45,9 +45,9 @@ _lint: $(LINTERS)
 .PHONY: fix $(FIXERS)
 fix: $(FIXERS)
 
-# END: lint-install ../bincapz
+# END: lint-install ../malcontent
 
-SAMPLES_REPO ?= chainguard-dev/bincapz-samples
+SAMPLES_REPO ?= chainguard-dev/malcontent-samples
 SAMPLES_COMMIT ?= b112eaf9bc547b2d219dfa775e0eff2874b30b9f
```

Comment:
I'll update this once https://github.com/chainguard-dev/bincapz-samples/pull/6 merges.